### PR TITLE
Reviewed and updated README files

### PR DESCRIPTION
**Pull request by ruthryi**
## Purpose
Reviewed and updated sample readme files for the next release

### CODE REVIEW

File: sample-scenarios/sample4/README.md [hunk 1]
```diff
@@ -36,21 +36,21 @@
...
+As and example to enable custom security algorithms we can engage Kerberos OAuth2 Grant by following this documentation(https://docs.wso2.com/display/AM220/Kerberos+OAuth2+Grant). [2]
 
 We can authorize the users through API Manager access tokens and we can use scopes to authorise the API consumers when consuming the APIS.
 
 By executing the scenario 4 using the run.sh as mentioned below will add the sample data relate the the above business use case to WSO2 API Manager distribution
 
-### How to run the sample to populate the above mentioned sample data
+### Running the sample
 
 1. Start wso2am-2.2.0-updateX is distribution by executing **[APIM_HOME]**/bin/wso2server.sh or **[APIM_HOME]**/bin/wso2server.bat
```
Comment (chamindias) [inline sample-scenarios/sample4/README.md]: Please replace all occurrences of "wso2am-2.2.0-updateX" with "wso2am-2.5.0"

File: sample-scenarios/sample4/README.md [hunk 2]
```diff
@@ -200,9 +200,9 @@ Further it will invoke the get with the scope and it will return the response as
 
 ### References
 
-[1] - [https://docs.wso2.com/display/AM2xx/Managing+Public%2C+Partner+vs+Private+APIs+Sample+Documentation](https://docs.wso2.com/display/AM2xx/Managing+Public%2C+Partner+vs+Private+APIs+Sample+Documentation)
+[1] - [https://docs.wso2.com/display/AM220/Managing+APIs](https://docs.wso2.com/display/AM220/Managing+APIs)
```
Comment (chamindias) [inline sample-scenarios/sample4/README.md]: This should be "AM250", isn't it?

File: sample-scenarios/sample1/README.md [hunk 3]
```diff
@@ -112,7 +112,7 @@ Below two screenshots shows that both John and TOm can subscribe to Employee_inf
 
 ### References
 
-[1] -[ https://docs.wso2.com/display/AM210/Key+Concepts#KeyConcepts-APIvisibilityandsubscription](https://docs.wso2.com/display/AM210/Key+Concepts#KeyConcepts-APIvisibilityandsubscription)
+[1] -[https://docs.wso2.com/display/AM220/Key+Concepts#KeyConcepts-APIvisibilityandsubscription](https://docs.wso2.com/display/AM220/Key+Concepts#KeyConcepts-APIvisibilityandsubscription)
```
Comment (chamindias) [inline sample-scenarios/sample1/README.md]: This should be "AM250", isn't it?

File: sample-scenarios/sample4/README.md (hunk 2 above)
Comment (ruthryi) [reply sample-scenarios/sample4/README.md]: We don't have docs for 2.5.0 yet. So adding the last GA release.

File: sample-scenarios/sample1/README.md (hunk 3 above)
Comment (ruthryi) [reply sample-scenarios/sample1/README.md]: Changed applicable instances found in the sample scenario README files.

File: sample-scenarios/sample4/README.md (hunk 2 above)
Comment (prasa7) [reply sample-scenarios/sample4/README.md]: No I think we have to put AM250. Otherwise when we release docs it should be a problem.

**Pull request merged**